scripts/run_frontend_tests.sh

```diff
@@ -41,6 +41,19 @@ source $(dirname $0)/setup_gae.sh || exit 1
 # TODO(sll): Make this work with fewer third-party dependencies.
 bash scripts/install_third_party.sh
 
+#build so as to have minified js and css
+echo ""
+echo "  Runing build task with concatenation only "
+echo ""
+
+$NODE_PATH/bin/node $NODE_MODULE_DIR/gulp/bin/gulp.js build
+
+echo ""
+echo "  Runing build task with concatenation and minification"
+echo ""
+
+$NODE_PATH/bin/node $NODE_MODULE_DIR/gulp/bin/gulp.js build --minify=True
```

Comment:
Oh, I see, OK. In that case, I have some suggestions for how this is dealt with in Karma; I'll make them above.